host/host.go

```diff
@@ -298,9 +297,9 @@ func runDaemon(args *docopt.Args) {
 		g.Log(grohl.Data{"at": "sampi_disconnected", "err": jobStream.Err})
 
 		// if the process is shutting down, just block
-		if sh.Active {
-			<-make(chan struct{})
-		}
+		//if shutdown.Active { // Wot?
```

Comment:
Sanity check, can we remove this?  I'm almost certain the other loop behavior continuing on here is fine.